### PR TITLE
Added LogOut Page

### PR DESCRIPTION
**Pull request by vipuluthaiah**
<div align="center">
  
</div>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,8 @@
 <img src="/images/1.png" height="400em" /> <img src="/images/2.png" height="400em" />
 <img src="/images/3.png" height="400em" /> <img src="/images/4.png" height="400em" />
 <img src="/images/5.png" height="400em" /> <img src="/images/6.png" height="400em" />
-<img src="/images/7.png" height="400em" />
+<img src="/images/7.png" height="400em" /> <img src="/images/8.jpg" height="400em" />
```
Comment (hiashutoshsingh) [inline README.md]: could you remove jpg file from here i think only gif is perfect and looks good.

**Review by hiashutoshsingh — COMMENTED** [review]
just a small change here @vipuluthaiah and we are set.

File: README.md (hunk 1 above)
Comment (vipuluthaiah) [reply README.md]: just a min...

**Review by vipuluthaiah — COMMENTED** [review]
Updated

**Comment by vipuluthaiah** [comment]
i am lil confused ..about the changes can u check it?

**Pull request merged**